### PR TITLE
Feat/signup

### PR DESCRIPTION
**Pull request by benhurfaria**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -3,29 +3,35 @@ import cors from 'cors';
...
     try {
-        res.status(201).send('ola');
+        res.status(200).send('ola');
     } catch (error) {
         console.log(error);
         res.sendStatus(500);
-
     }
 }
 );
```
Comment (Audamon) [inline src/app.js]: remover

File: src/app.js [hunk 2]
```diff
@@ -3,29 +3,35 @@ import cors from 'cors';
...
+        res.status(200).send('ola mundo');
+    } catch (error) {
+        console.log(error);
+        res.sendStatus(500);
+    }
+});
 
+app.get('/xablau', (req,res)=>{
+    res.sendStatus(500);
+})
```
Comment (Audamon) [inline src/app.js]: remover

File: src/tests/signup.test.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+afterAll(async ()=>{
+    connection.end();
+});
+
+describe("POST /sign-up", ()=>{
+    const validBody = validBodyFactorySignup();
+    const invalidBody = invalidBodyFactorySignup();
+    
+    afterAll(async () =>{
+        console.log(validBody.email);
```
Comment (Audamon) [inline src/tests/signup.test.js]: remover o console.log

File: src/tests/signup.test.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        `);
+    });
+    
+    console.log(validBody.email);
+    test("returns 400 there is some inconsistent information", async ()=>{
+        const result = await supertest(app).post('/sign-up').send(invalidBody);
+
+        expect(result.status).toEqual(400);
+    });
+    console.log(validBody.email);
```
Comment (Audamon) [inline src/tests/signup.test.js]: remover console.log

File: src/tests/signup.test.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    test("returns 200 when send valid information", async ()=>{
+        const result = await supertest(app).post('/sign-up').send(validBody);
+
+        expect(result.status).toEqual(200);
+    });
+    console.log(validBody.email);
```
Comment (Audamon) [inline src/tests/signup.test.js]: remover console.log

**Review by Audamon — CHANGES_REQUESTED** [review]

**Review by FranciscaFigueiredo — APPROVED** [review]

**Review by Audamon — APPROVED** [review]

**Pull request merged**